src/index.js

```diff
@@ -11,10 +11,11 @@ function defaultAssignEntity(normalized, key, entity) {
 function visitObject(obj, schema, bag, options) {
   const { assignEntity = defaultAssignEntity } = options;
 
-  let normalized = {};
+  const defaults = schema && schema.getDefaults && schema.getDefaults();
+  const schemaAssignEntity = schema && schema.getAssignEntity && schema.getAssignEntity();
+  let normalized = isObject(defaults) ? defaults : {};
```

Comment:
Good catch. My bad (( didn't think about object reference. I'll go with spread, because as I understand Object.assign should be polyfilled.